### ISSUE TITLE
Select organization as part of invitation flow

### ISSUE THREAD

**Issue by bjarnef** [umbraco/Umbraco.Cloud.Issues#750]
### Issue description

Often we need to invite Umbraco HQ SWAT team members to Umbraco Cloud project:
I think we can make this a bit simpler.

The invite could perhaps have a dropdown before name (default set to your organization)
- Umbraco HQ
- My organization

Which update the datalist suggestions next... which fill out name, email etc.

It is still possible to invite any other user.

Perhaps the Umbraco HQ organization should just contain SWAT team, because it is those we typically need to invite.

**Comment by ksuvec** [comment]
hi @bjarnef 

Am I understanding this correctly that you want the possibility to have a dropdown for Umbraco HQ members available everytime you invite a user to your project?

**Comment by bjarnef** [comment]
@ksuvec yes something like this to select organization first, which could be our organization ny default, but we could switch to Umbraco and select a SWAT member.. I imagine in update the datalist or has a second datalist.

> The invite could perhaps have a dropdown before name (default set to your organization)

Umbraco HQ
My organization

Selection just populate name and email, but any other from Umbraco or other user can still be invited.

Just think it could make it a bit simpler to select "Anders" or someone else from SWAT and no need to selct admin etc (or could set til default and perhaps disabled specific for Umbraco HQ)?

**Comment by bjarnef** [comment]
I would also suggest this remove the suffix of company added to name in datalist :)

**Comment by bjarnef** [comment]
@ksuvec would it be possible to show e.g. a dropdown first:

**Organization**
- Umbraco HQ
- My organization
- Other (or just ensure it is possible to invite any).

Selecting "Umbraco HQ" shows a combobox of employees and easy to search for name.
https://uui.umbraco.com/?path=/story/uui-combobox--avatars

And hide/set the rest based on employee, because HQ support always ask to set these :)

<img width="567" height="1047" alt="Image" src="https://github.com/user-attachments/assets/31608561-81cb-4811-954b-30406824b382" />

From chat we also only see first name (but I prefer to enter full name), so I tend to find the employee here:
https://umbraco.com/about-us/umbraco-hq-team/

**Comment by mikkelhm** [comment]
Im closing this issue down - We are not going to prioritize this issue - it a neat helper, but only if you are in search for inviting someone form Umbraco - most likley during a support session. This is not the intended workflow, and would then be shown everytime someone invites a person to their project - which in most cases should not involve Umbraco HQ members.

**Comment by bjarnef** [comment]
@mikkelhm not sure what the intended workflow is, but with many Umbraco Cloud projects, we once in while need to investigate issues with the support team, and get the same message each time how to invite.
We also prefer to add full name, so we find the employee at the Umbraco HQ page.
There must be a simpler way to handle time (or perhaps directly from the chat?).

Another part of this is to invite developers from our own organization, which is almost the same developers from our team (registered under our organization in the portal).
It is fine to be able to invite one by one, but in most cases internally it would be a time saver to just pick multiple developer (thus the suggestion to select organization, and next pick multiple developers, which always need admin access).

Of course we also invite project manager and customer, but not that often as developers.
